packages/channels/src/teams/stream.ts

```diff
@@ -32,9 +32,14 @@ export class TeamsStream extends ChannelStream<TeamsService, TeamsContext> {
         grant_type: 'client_credentials',
         client_id: config.appId,
         client_secret: config.appPassword,
+        tenant_id: config.tenantId,
         scope: 'https://api.botframework.com/.default'
       })
-      await axios.post('https://login.microsoftonline.com/botframework.com/oauth2/v2.0/token', params.toString())
+      if (config.tenantId) {
+        await axios.post(`https://login.microsoftonline.com/${config.tenantId}/oauth2/v2.0/token`, params.toString())
+      } else {
+        await axios.post('https://login.microsoftonline.com/botframework.com/oauth2/v2.0/token', params.toString())
+      }
```

Comment:
That's literally the same logic, here's my suggestion: 
```suggestion
const tenant = config.tenantId ?? 'botframework.com'
await axios.post(`https://login.microsoftonline.com/${tenant}/oauth2/v2.0/token`, params.toString())
```

Reply:
Even better thanks